docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -63,6 +63,35 @@ status: {}
 More information about the [scale subresource can be found in the Kubernetes
 documentation][scale].
 
+#### Required `spec` fields for implementations using version
+
+* `version` - is a string representing the Kubernetes version to be used
+  by the control plane machines. The value must be a valid semantic version;
+  also if the value provided by the user does not start with the v prefix, it
+  should be added.
```

Comment:
@fabriziopandini Should or must? Just because if it's a should, we must handle both cases.

Reply:
fixed

Reply:
In case we want to fix this sooner or later it might be worth starting a tracking issue for breaking API changes for the next breaking release